client/app/vps/dashboard/vps-dashboard.controller.js

```diff
@@ -94,6 +97,40 @@ class VpsDashboardCtrl {
             .catch(() => { this.hasAdditionalDisk = false });
     }
 
+    snapshotOption () {
+        this.snapshotDescription = this.summary.snapshot.creationDate ?
+            this.$translate.instant("vps_tab_SUMMARY_snapshot_creationdate") + " " + moment(this.summary.snapshot.creationDate).format("LLL") :
```

Comment:
String templating instead. You could also use `` this.summary.snapshot.creationDate | momentFormat: "LLL" ``